pkgs/tools/security/aeskeyfind/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aeskeyfind";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://citpsite.s3.amazonaws.com/memory-content/src/aeskeyfind-${version}.tar.gz";
+    sha256 = "sha256-FBflwbYehruVJ9sfW+4ZlaDuqCR12zy8iA4Ev3Bgg+Q=";
+  };
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp aeskeyfind $out/bin
+    runHook postInstall
+  '';
+  meta = with lib; {
+    description = "Locates 128-bit and 256-bit AES keys in a captured memory image.";
```

```suggestion
    description = "Locates 128-bit and 256-bit AES keys in a captured memory image";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes